README.md

```diff
@@ -240,6 +240,30 @@ Unlike the other libraries supported here, JAX array API support is contained
 entirely in the JAX library. The JAX array API support is tracked at
 https://github.com/google/jax/issues/18353.
 
+## Dask
+
+If you're using dask with numpy, many of the same limitations that apply to numpy
+will also apply to dask. Besides those differences, other limitations include missing
+sort functionality (no `sort` or `argsort`), and limited support for the optional `linalg`
+and `fft` extensions.
+
+In particular, the `fft` namespace is not compliant with the array API spec. Any functions
```

Comment:
Out of interest, is the plan to implement `array_api_compat.dask.{fft, linalg}` or wait for support from `dask` itself? A similar question w.r.t JAX.

Reply:
I haven't attempted to wrap fft yet - waiting on #78 to do so. 

Linalg can only be partially supported by us since there's missing methods in dask.